server/index.js

```diff
@@ -31,4 +31,62 @@ app.get('/products/:product_id', (req, res) => {
    })
 })
 
-// TODO: app.get('/products/reviews/:product_id') => https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/reviews?product_id=37312
\ No newline at end of file
+// return product's related product array at endpoint id's 'related content'
+app.get('/products/:product_id/related', (req, res) => {
```

Comment:
does :product_id dynamically render? or do we need /products/{product_id}/related